src/Grpc.AspNetCore.Server/Internal/Constants.cs

```diff
@@ -0,0 +1,29 @@
+﻿#region Copyright notice and license
+
+// Copyright 2019 The gRPC Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#endregion
+
+using Grpc.Core;
+using Microsoft.Extensions.Primitives;
+
+namespace Grpc.AspNetCore.Server.Internal
+{
+    internal static class Constants
+    {
+        public const string GrpcStatusHeader = "grpc-status";
```

Comment:
nit: this is always sent in a trailer, so GrpcStatusTrailer is probably a better name.